packages/babel-cli/src/babel/info.js

```diff
@@ -0,0 +1,18 @@
+import { run } from "envinfo";
+
+export default async function() {
+  console.log("\nEnvironment Info:");
+  return run(
+    {
+      System: ["OS", "CPU"],
+      Binaries: ["Node", "npm", "Yarn"],
+      Browsers: ["Chrome", "Edge", "Firefox", "Safari"],
+      npmPackages: "/**/{*babel*,@babel/*/}",
+      npmGlobalPackages: "*babel*",
+    },
+    {
+      duplicates: true,
+      showNotFound: true,
+    },
+  ).then(console.log);
```

Comment:
Doesn't console.log throw when called without a `this` object?

Reply:
I've never seen any issue from this - I use it all the time, and this is also how it's implemented in CRA.